hera_mc/geo_handling.py

```diff
@@ -23,7 +23,7 @@
 from hera_mc import mc, part_connect, cm_utils, geo_location, cm_revisions
 
 
-def cofa(show_cofa=False, session=None):
+def cofa(session=None):
```

Comment:
The docstring description of the now-gone `show_cofa` argument should be removed.